src/styles/header.scss

```diff
@@ -0,0 +1,36 @@
+@import "utils.scss";
```

Comment:
there is a warning for Deprecated Sass @import rules , kindly swap @import with @use to be able to include mixins and global styling 
```@use "./utils.scss" as *; ```

Reply:
fixed!